### ISSUE TITLE
Package disabes column sorting after third attempt.

### ISSUE THREAD

**Issue by preliot** [Flagstudio/nova-actions-left#2]
This package disables column sorting after the third try. First i tought it was a laravel Nova issues [https://github.com/laravel/nova-issues/issues/2503](https://github.com/laravel/nova-issues/issues/2503), but this morning i had an epiphany.

In `resources/views/vendor/nova/layout.blade.php` i've inserted all the scripts manually (and removed the foreach for tool scripts). When i removed this script, table sorting worked as expected.

```javascript
//...
<!--<script src="/nova-api/scripts/nova-actions-left"></script>-->
//...

```
Recently the laravel nova team added the option to remove table sorting on the third click. Perhaps that will help?

Laravel Version: 5.8.37
Nova Version: 2.11.1
PHP Version: 7.2.19
Operating System and Version: Windows 10
Browser type and version: Chrome 79 64-bits

and also

Laravel Version: 7.11.0
Nova Version: 3.6.0
PHP Version: 7.2.19
Database Driver & Version: mysqlm 5.7.24
Operating System and Version: windows 10
Browser type and version: chrome 81 (64 bits)

**Comment by vasishakd** [comment]
Fixed in release [v0.2.0](https://github.com/Flagstudio/nova-actions-left/releases/tag/v0.2.0) for laravel/nova version `2.10.0` and higher

**Comment by preliot** [comment]
Thanks for your quick support, it is now working like a charme!
